tests/scripts/thread-cert/Cert_7_1_07_BorderRouterAsLeader.py

```diff
@@ -0,0 +1,567 @@
+                   (_pkt.mle.tlv.leader_data.stable_data_version + 1) % 256
+                   ).\
+            must_next()
+        with pkts.save_index():
+            _dr_pkt1 = pkts.filter_wpan_src64(LEADER).\
+                filter_LLANMA().\
+                filter_mle_cmd(MLE_DATA_RESPONSE).\
+                filter(lambda p: {
+                                  NETWORK_DATA_TLV,
+                                  SOURCE_ADDRESS_TLV,
+                                  LEADER_DATA_TLV
+                                 } <= set(p.mle.tlv.type) and\
+                                 {
+                                  NWD_BORDER_ROUTER_TLV,
+                                  NWD_6LOWPAN_ID_TLV
+                                 } <= set(p.thread_nwd.tlv.type) and\
+                       p.mle.tlv.leader_data.data_version ==
+                       (_dr_pkt.mle.tlv.leader_data.data_version + 1) % 256 and\
+                       p.mle.tlv.leader_data.stable_data_version ==
+                       _dr_pkt.mle.tlv.leader_data.stable_data_version and\
+                       sublist([ROUTER_1_RLOC16, ROUTER_2_RLOC16],
+                               p.thread_nwd.tlv.border_router_16) and\
+                       sublist([0, 1, 1, 1, 0], p.thread_nwd.tlv.stable) and\
+                       sublist([1], p.thread_nwd.tlv.__getattr__('6co.flag.c')) and\
+                       sublist([Ipv6Addr(PREFIX_1[:-3])], p.thread_nwd.tlv.prefix)
+                       ).\
+                must_next()
+
+        # Step 6: Leader MUST send a MLE Child Update Request or MLE Data
+        #         Response to SED, including the following TLVs:
+        #             - Network Data TLV
+        #                 At least one Prefix TLV (Prefix 1) including:
+        #                     - Stable Flag set
+        #                     - Border Router sub-TLV(corresponding to Router_1)
+        #                         - P_border_router_16<0xFFFE>
+        #                         - Stable Flag set
+        #                     - 6LoWPAN ID sub-TLV
+        #                         - Stable Flag set
+        #             - Source Address TLV
+        #             - Leader Data TLV
+        #             - Active Timestamp TLV
+        pkts.filter_wpan_src64(LEADER).\
+            filter_wpan_dst64(SED).\
+            filter_mle_cmd2(MLE_CHILD_UPDATE_REQUEST, MLE_DATA_RESPONSE).\
+            filter(lambda p: {
+                              NETWORK_DATA_TLV,
+                              SOURCE_ADDRESS_TLV,
+                              LEADER_DATA_TLV,
+                              ACTIVE_TIMESTAMP_TLV
+                             } == set(p.mle.tlv.type) and\
+                   sublist([Ipv6Addr(PREFIX_1[:-3])], p.thread_nwd.tlv.prefix) and\
+                   sublist([1, 1, 1], p.thread_nwd.tlv.stable) and\
+                   sublist([1], p.thread_nwd.tlv.__getattr__('6co.flag.c')) and\
```

Comment:
```suggestion
                   sublist([1], getattr(p.thread_nwd.tlv, '6co').flag.c)) and\
```
I didn't realize that `__getattr__('6co.flag.c')` can also work.